engine/distconsts/consts.go

```diff
@@ -16,3 +16,11 @@ const (
 	PythonSDKManifestDigestEnvName     = "DAGGER_PYTHON_SDK_MANIFEST_DIGEST"
 	TypescriptSDKManifestDigestEnvName = "DAGGER_TYPESCRIPT_SDK_MANIFEST_DIGEST"
 )
+
+const (
+	AlpineVersion = "3.20.0"
+	AlpineImage   = "alpine:" + AlpineVersion
+
+	GolangVersion = "1.22.3"
```

Comment:
Should we bump this to `1.22.4` so that we benefit from the [latest security fixes](https://github.com/golang/go/issues?q=milestone%3AGo1.22.4+label%3ACherryPickApproved)?

Reply:
Done!